src/java.base/share/lib/security/default.policy

```diff
@@ -226,7 +226,6 @@ grant codeBase "jrt:/openj9.cuda" {
     permission java.lang.RuntimePermission "loadLibrary.cuda4j29";
     permission java.lang.RuntimePermission "modifyThread";
     permission java.lang.RuntimePermission "modifyThreadGroup";
-    permission java.util.PropertyPermission "com.ibm.oti.vm.library.version", "read";
```

Comment:
I don't think this should be removed. Access has always been allowed to this property, removing the access would break customers who are already using it.

Reply:
Nm, I should have looked at the context.